lib/vanagon/platform.rb

```diff
@@ -226,7 +230,7 @@ def is_unix?
     #
     # @return [true, false] true if it is a windows variety, false otherwise
     def is_windows?
-      return !!@name.match(/^win-.*$/)
+      return !!@name.match(/^windows-.*$/)
```

Comment:
Rubocop is weird about this. It suggests using `!name.match(/^windows-.*$/).nil?` instead. I have mixed feelings, as the double-negation is common enough in our codebase. Does anyone have thoughts about this?

(I know that this isn't the PR to fix this stuff in, but we might as well get a list together).